src/static/js/ethers_helper.js

```diff
@@ -1944,7 +1944,14 @@ function getCurvePrices(prices, pool) {
   var staked_tvl = pool.staked * price;
   const poolUrl = `https://etherscan.io/token/${pool.address}`;
   const name = `<a href='${poolUrl}' target='_blank'>${pool.symbol}</a>`;
-  const dexguruTokenlink = `<a href='https://dex.guru/token/${pool.address}-${chain}' noopener noreferrer target='_blank'>TradingView Chart</a>`;
+  const dexguruTokenlink =  function() {
```

Comment:
this is function, not variable. so here below `_print(`${name} Price: $${formatMoney(price)} Market Cap: $${formatMoney(tvl)} ${dexguruTokenlink}`);` it wouldn't work